src/overlays/actors/ovl_En_Bba_01/z_en_bba_01.c

```diff
@@ -33,84 +42,296 @@ const ActorInit En_Bba_01_InitVars = {
+    /* Sword          */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Goron pound    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Fire arrow     */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Ice arrow      */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Light arrow    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Goron spikes   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Deku spin      */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Deku bubble    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Deku launch    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* UNK_DMG_0x12   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Zora barrier   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Normal shield  */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Light ray      */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Thrown object  */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Zora punch     */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Spin attack    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Sword beam     */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Normal Roll    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* UNK_DMG_0x1B   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* UNK_DMG_0x1C   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Unblockable    */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* UNK_DMG_0x1E   */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
+    /* Powder Keg     */ DMG_ENTRY(0, BBA01_DMG_EFF_NONE),
 };
 
-#endif
+u16 D_809CCCD8[] = { 4000, 4, 1, 3, 6000, 4, 1, 6, 4000, 4, 1, 3, 6000, 4, 1, 6 };
+
+void EnBba01_UpdateModel(EnBba01* this, GlobalContext* globalCtx) {
+    Player* player = GET_PLAYER(globalCtx);
+    Vec3f focus;
+
+    EnHy_UpdateSkelAnime(&this->enHy, globalCtx);
+    if (func_8013D5E8(this->enHy.actor.shape.rot.y, 0x36B0, this->enHy.actor.yawTowardsPlayer)) {
+        focus.x = player->actor.world.pos.x;
+        focus.y = player->bodyPartsPos[7].y + 3.0f;
+        focus.z = player->actor.world.pos.z;
+        func_8013D2E0(&focus, &this->enHy.actor.focus.pos, &this->enHy.actor.shape.rot, &this->enHy.focusTarget,
+                      &this->enHy.headRot, &this->enHy.torsoRot, D_809CCCD8);
+    } else {
+        Math_SmoothStepToS(&this->enHy.focusTarget.x, 0, 4, 0x3E8, 1);
+        Math_SmoothStepToS(&this->enHy.focusTarget.y, 0, 4, 0x3E8, 1);
+        Math_SmoothStepToS(&this->enHy.headRot.x, 0, 4, 0x3E8, 1);
+        Math_SmoothStepToS(&this->enHy.headRot.y, 0, 4, 0x3E8, 1);
+        Math_SmoothStepToS(&this->enHy.torsoRot.x, 0, 4, 0x3E8, 1);
+        Math_SmoothStepToS(&this->enHy.torsoRot.y, 0, 4, 0x3E8, 1);
+    }
+    SubS_FillLimbRotTables(globalCtx, this->enHy.limbRotTableY, this->enHy.limbRotTableZ,
+                           ARRAY_COUNT(this->enHy.limbRotTableY));
+    EnHy_UpdateCollider(&this->enHy, globalCtx);
+}
+
+s32 EnBba01_TestIsTalking(EnBba01* this, GlobalContext* globalCtx) {
+    s32 isTalking = false;
+
+    if (Actor_ProcessTalkRequest(&this->enHy.actor, &globalCtx->state)) {
+        isTalking = true;
+        this->enHy.textId = 0x10B9;
```

Comment:
```suggestion
        this->enHy.textId = 0x10B9; // invalid textId, produces empty textbox
```
Optional: I like textIds to be documented, in this case I spawned her and she produces an empty textbox that ends with a blue square conclusion icon, nothing interesting.

[Zelda Decomp Discord Video Link] https://cdn.discordapp.com/attachments/688852030403379277/935281291472035860/weird_en_bba_shadow.mp4

I just don't want people checking this actor out of curiosity to see this textId thinking it might have beta dialogue and go on a goose chase for an empty textbox.